projects/plugins/jetpack/class.jetpack.php

```diff
@@ -5413,9 +5418,9 @@ public function verify_json_api_authorization_request( $environment = null ) {
 		}
 
 		if ( ! $signature ) {
-			wp_die( $die_error );
+			wp_die( esc_html( $die_error ) );
```

Comment:
Line 5393 is producing HTML that shouldn't be escaped here. Kses it maybe?

Reply:
Shoulda got all these in 5d977fb4d2b05de23fe7ba4a4ab3e1beff7cb64a, let me know if that looks okay.